shared/js/dialer/keypad.js

```diff
@@ -1,8 +1,9 @@
 /* exported KeypadManager */
```

Comment:
This is a good point. I'd say we should file a followup to disable this later. AOSP just doesn't present the user with a # symbol in their Emergency Dialer.